src/plots/cartesian/axes.js

```diff
@@ -1792,7 +1795,9 @@ axes.doTicks = function(gd, axid, skipTitle) {
         }
         else {
             flipit = (axside === 'right') ? 1 : -1;
-            labely = function(d) { return d.dy + d.fontSize / 2 - labelShift * flipit; };
+            labely = function(d) {
+                return d.dy + d.fontSize * 0.35 - labelShift * flipit;
```

Comment:
Non- :no_entry: (might be suited for #1434 - the part where we proposed making it configurable) but this `0.35` would look really nice side-by-side [`LINE_SPACING`](https://github.com/plotly/plotly.js/blob/master/src/constants/alignment.js#L33).